connectors/src/resources/zendesk_resources.ts

```diff
@@ -273,4 +337,138 @@ export class ZendeskCategoryResource extends BaseResource<ZendeskCategory> {
       where: { connectorId, categoryId, permission: "read" },
     });
   }
+
+  toContentNode({ connectorId }: { connectorId: number }): ContentNode {
+    return {
+      provider: "zendesk",
+      internalId: getCategoryInternalId(connectorId, this.categoryId),
+      parentInternalId: getHelpCenterInternalId(connectorId, this.brandId),
+      type: "folder",
+      title: this.name,
+      sourceUrl: this.url,
+      expandable: false,
+      permission: this.permission,
+      dustDocumentId: null,
+      lastUpdatedAt: this.updatedAt.getTime(),
+    };
+  }
+}
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+// eslint-disable-next-line @typescript-eslint/no-unsafe-declaration-merging
+export interface ZendeskTicketResource
+  extends ReadonlyAttributesType<ZendeskTicket> {}
+
+// eslint-disable-next-line @typescript-eslint/no-unsafe-declaration-merging
+export class ZendeskTicketResource extends BaseResource<ZendeskTicket> {
+  static model: ModelStatic<ZendeskTicket> = ZendeskTicket;
+
+  constructor(
+    model: ModelStatic<ZendeskTicket>,
+    blob: Attributes<ZendeskTicket>
+  ) {
+    super(ZendeskTicket, blob);
+  }
+
+  async postFetchHook(): Promise<void> {
+    return;
+  }
```

Comment:
is it needed ?

Reply:
yes, the method is abstract in the base class

Reply:
indeed